docs/src/main/apps/cswclusterseed.md

```diff
@@ -21,11 +21,11 @@ Note: An alternative method is to run `sbt publishLocal stage`, which installs a
 Unzip either the downloaded or constructed zip file and switch current
 working directory to the extracted folder. Choose appropriate instructions from below based on requirement ie single seed or multiple seeds.
  
-# Environment Variables
+## Environment Variables
```

Comment:
By changing the heading of "Environment variables" I am afraid it won't appear in "Table of contents" at right side.

Reply:
Seems like a correct change to us